modules/nw-throughput-troubleshoot.adoc

```diff
@@ -0,0 +1,53 @@
+// Module filename: nw-throughput-troubleshoot.adoc
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id='nw-throughput-troubleshoot-{context}']
+= Troubleshooting Throughput Issues
+Sometimes applications deployed through {product-title} can cause
+network throughput issues such as unusually high latency between
+specific services.
+
+Use the following methods to analyze performance issues if pod logs do not
+reveal any cause of the problem:
+
+* Use a packet analyzer, such as ping or link:http://www.tcpdump.org/[tcpdump]
+to analyze traffic between a pod and its node.
++
+For example, run the tcpdump tool on each pod while reproducing the behavior
+that led to the issue.
+Review the captures on both sides to compare send and receive timestamps to
+analyze the latency of traffic to/from a pod.
+Latency can occur in {product-title} if a node interface is overloaded with
+traffic from other pods, storage devices, or the data plane.
++
+----
+$ tcpdump -s 0 -i any -w /tmp/dump.pcap host <podip 1> && host <podip 2> <1>
+----
++
+<1> `podip` is the IP address for the pod. Run the following command to get
+the IP address of the pods:
++
+----
+# oc get pod <podname> -o wide
```

Comment:
Is there a reason to run this as root (because of "#") or can it be changed to use "$"?